Registry/RegistryParse.py

```diff
@@ -238,14 +238,31 @@ def __init__(self, buf, offset, parent):
         if _id != 0x66676572:
             raise ParseException("Invalid REGF ID")
 
-        _seq1 = self.unpack_dword(0x4)
-        _seq2 = self.unpack_dword(0x8)
+    def hive_sequence1(self):
+        """
+        Get first sequence number.
+        Believed to be saved when a commit is started.
+        """
+        return self.unpack_dword(0x4)
+
+    def hive_sequence2(self):
+        """
+        Get second sequence number.
+        Believed to be saved to the same value as squence1 when a commit is completed.
+        """
+        return self.unpack_dword(0x8)
 
-        if _seq1 != _seq2:
-            # the registry was not synchronized
-            pass
+    def valid_sequence(self):
+        """
+        Compare if sequence numbers is correct.
+        """
+        return self.hive_sequence1() == self.hive_sequence2()
 
-        # TODO: compute checksum and check
+    def modification_timestamp(self, offset):
```

Comment:
That should not have been there, not sure it is correct, needs some testing.
at least offset argument should go away. ;)